tst/schedulers/test_grid_searcher.py

```diff
@@ -102,107 +94,6 @@ def test_store_and_restore_state_with_initial_config():
         assert new_config in all_candidates_on_grid
 
 
-def test_grid_scheduler_continuous():
```

Comment:
In the PR I just merged in, these two tests are run with simulator back-end, so they are quite fast then. We'd not have to remove them.

Reply:
Also, you'd need to check `tst/test_schedulers.py`, where we now also run things with simulator back-end, for a longer wallclock time. Some of these cases may be >1s. Marking them with 5s (for each case, I suppose?) should be fine.

Reply:
Thanks for the pointer for the grid search, it is indeed very fast now. Regarding the schedulers test, I removed the BO tests for now as they take currently 3 min to run.